File: JavaScript/프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    var answer = n;
    for(var i=0; i<control.length;i++){
        if(control.charAt(i)=='w') answer+=1; 
        else if(control.charAt(i)=='s') answer-=1; 
        else if(control.charAt(i)=='d') answer+=10; 
        else if(control.charAt(i)=='a') answer-=10; 
    }
    return answer;
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.cs
using System;

public class Solution {
    public int solution(int n, string control) {
        int answer = n;
        for(int i=0; i<control.Length;i++){
            if(control[i] == 'w') answer+=1; 
            if(control[i] == 's') answer-=1; 
            if(control[i] == 'd') answer+=10; 
            if(control[i] == 'a') answer-=10; 
        }
        return answer;
    }
}